rest-service/manager_rest/test/mock_blueprint/blueprint_with_default_schedules.yaml

```diff
@@ -12,6 +17,8 @@ deployment_settings:
       recurrence: 1w
       weekdays: [mo, fr]
       count: 5
+      workflow_parameters:
```

Comment:
funnily, while parameters can use intrinsic functions indeed, other things not necessarily;  this is because parameters are declared (dsl-parser side) as a dict, and so they are considered for evaluating functions. But the other things (eg. workflow id) are just declared as a string, so it's invalid to give them a get-input.